scripts/install-lib

```diff
@@ -55,3 +69,21 @@ how-to-enable() {
 
     echo "$ systemctl enable --now ${1/.service/}"
 }
+
+# Reload application registrations in oxide
+#
+# Output:
+#
+# Reload status message
+reload-oxide-apps() {
```

Comment:
The one concern I have with this on every install/upgrade/remove of an application, is that if you install multiple applications, you only need to call it once, instead of once per application. So extra processing happens. Not really sure how to work around that though.

Reply:
Well now that I've tested, it's rather interesting watching them disappear one by one and then reappear all at once. Doesn't seem to be too big of a deal.